src/inspector_profiler.cc

```diff
@@ -121,21 +121,26 @@ static bool EnsureDirectory(const std::string& directory, const char* type) {
   return true;
 }
 
-std::string V8CoverageConnection::GetFilename() const {
+std::string V8CoverageConnection::GetFilename(const char* prefix) const {
   std::string thread_id = std::to_string(env()->thread_id());
   std::string pid = std::to_string(uv_os_getpid());
   std::string timestamp = std::to_string(
       static_cast<uint64_t>(GetCurrentTimeInMicroseconds() / 1000));
   char filename[1024];
```

Comment:
just a nit for later... should look into whether this can be replaced with `DiagnosticFilename` at some point

Reply:
sounds like this refactor doesn't need to block this PR?

Reply:
I've combined the source-map and coverage JSON in my latest pass at this PR.